packages/core/src/generatePolicy.js

```diff
@@ -116,7 +120,10 @@ function createModuleInspector(opts) {
     if (!packageToNativeModules.has(packageName)) {
       packageToNativeModules.set(packageName, [])
     }
-    const packageNativeModules = packageToNativeModules.get(packageName)
+    const packageNativeModules =
+      /** @type {import('./moduleRecord').LavamoatModuleRecord[]} */ (
```

Comment:
- [ ] replace these with `@import`s

Reply:
I just ripped out all changes to `packages/core/src/generatePolicy.js` since I reverted my original changes and the others had nothing to do with what I need here